### PR TITLE
FR Update

### PR DESCRIPTION
**Pull request by Buchodt**
FR Update by Bucho

### CODE REVIEW

File: EDEngineer/Resources/Data/localization.json [hunk 1]
```diff
@@ -1952,144 +1952,6 @@
       "de": "Bestandteil in keinem bekannten Bauplan!",
       "fr": null,
       "ru": null
-    },
```
Comment (Buchodt) [inline EDEngineer/Resources/Data/localization.json]: Sry :) Did not on purpose xD

Comment (msarilar) [inline EDEngineer/Resources/Data/localization.json]: you need to fix the file by putting back what you removed ; I'll accept the PR after :)

File: EDEngineer/Resources/Data/localization.json [hunk 2]
```diff
@@ -174,55 +174,55 @@
     "Low Emissions ": {
       "es": "Bajas Emisiones ",
       "de": "Emissionsoptimiertes ",
-      "fr": null,
+      "fr": "Basse émission"F,
```
Comment (msarilar) [inline EDEngineer/Resources/Data/localization.json]: think you need to remove that 'F' too

File: EDEngineer/Resources/Data/localization.json (hunk 1 above)
Comment (Buchodt) [inline EDEngineer/Resources/Data/localization.json]: I dont understand how i removed that xD And how i put it back xD Sry ! Trying ...

Comment (msarilar) [inline EDEngineer/Resources/Data/localization.json]: copy paste :)
edit the file with a text editor then commit and push your change:

```
git commit -am"added back what I removed, and removed pesky F typo"
git push origin patch-1
```

**Comment by Buchodt** [comment]
I dont really understood what i did xD, is it Ok ? xD

**Comment by msarilar** [comment]
no there was conflicts, I fixed them myself
the explanation:

1) You forked
2) I added the synthesis localization
3) You merged back but you deleted everything for the synthesis
4) A german translator submitted a PR for the synthesis localization
5) You added back the synthesis localization but without the german localization recently added
6) conflict !

it was very messy to solve so I did it with beyondcompare which solved it automatically (I don't think you have such tool). next time when you merge back the main repo to your fork, try not to erase the changes, because that's the source of all this :o

but now it's merged in master and will be available next deploy

**Comment by msarilar** [comment]
also let's not put crap commit message :) I did not check the commit message before accepting the PR

`Update my sh*t xD`
https://github.com/msarilar/EDEngineer/commits/master

I won't bother reverting and rebasing to generate a new commit for that one (you are never supposed to change history in git) but that's just for the future

**Pull request merged**